modules/rosa-list-objects.adoc

```diff
@@ -176,6 +175,42 @@ List all API and ingress endpoints for a cluster named `mycluster`:
 $ rosa list ingresses --cluster=mycluster
 ----
 
+== list instance-types
+
+List all of the available instance types for use with {product-title}. Availability is based on the account's AWS quota.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list instance-types [arguments]
+----
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--output
+|The output format. Allowed formats are [json \| yaml].
```

Comment:
Although `[json | yaml]` is the syntax usually shown for actual commands (like how you do [arguments] above), I don't think that's normally how we would write it in a normal sentence. Suggest something like:

```
Allowed formats are `json` or `yaml`.
```